tailwind.config.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
       events: "url('/assets/backgrounds/events.jpg')",
       contact: "url('/assets/backgrounds/contact.jpg')",
       aksaraBox: "url('/assets/backgrounds/aksaraBox.png')",
+      profile: "url('/assets/backgrounds/profile.jpg')",
```

Comment:
you can try using aksara format since you are using alt, so you dont need to define a new shortcut path to this profile